packages/core/src/extensions/renderer-api/components.ts

```diff
@@ -72,14 +75,14 @@ export * from "../../renderer/components/line-progress";
 export * from "../../renderer/components/menu";
 
 export {
-  NotificationStatus,
+  type NotificationStatus,
```

Comment:
I don't think we should change this export since it is a typescript enum which can be used in expressions.